src/ports/postgres/modules/lda/lda.py_in

```diff
@@ -445,6 +511,12 @@ def lda_train(schema_madlib, train_table, model_table, output_data_table, voc_si
             'invalid argument: positive real expected for alpha')
     _assert(beta is not None and beta > 0,
             'invalid argument: positive real expected for beta')
+    _assert(evaluate_every is not None and evaluate_every >= 0,
```

Comment:
The user docs for evaluate_every mention `Set it to 0 or negative number to not evaluate perplexity in training at all` but this check will throw an exception for evaluate_every < 0

Reply:
I have removed this check as we are not calculating the perplexity for 0 or -1.